azure-pipelines/vars/snd1.yaml

```diff
@@ -0,0 +1,14 @@
+variables:
+  environment: snd 
+  subscriptionName: AZD-CDO-SND1
+  ssvAppRegServicePrincipalId: b657d032-1be7-492f-8693-15180569c703 #ADO-DefraGovUK-AAD-CDO-SSV3
+    
+  # KeyVault
+  keyVaultName: SSVCDOINFKV3401
+  keyVaultResourceGroup: SSVCDOINFRG3401
+  keyvaultEnableSoftDelete: false
+  keyvaultEnablePurgeProtection: false
```

Comment:
These variables have the same values as common... probably don't need it...
Same thing for SND2 and SND3

Reply:
@ajaysoni-cap : Yes. These will be same for SND1,SND2,SND3 but it will be different in Defra tenant.
The reason to override it in environment variable file is to keep changing variables and its order consistent in all environment file so it is easy to maintain and read as well.